index.html

```diff
@@ -5,22 +5,58 @@
     	<meta name="description" content="Your description goes here">
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
-
+		<title>Relaxr</title>
+		<!-- Google fonts -->
+		<link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,800' rel='stylesheet' type='text/css'>
 		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/normalize.css">
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
 	<body>
 		<div id="container">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<section class="content-width">
+					<h1 class="shift-left">Relaxr</h1>
+					<h2>Get piece of mind with a single tap</h2>
+					<a href="#">Get it Now</a>
+				</section>
 			</header>
+			<main>
+				<section class="content-width">
+					<h3 class="shift-left">Benefits</h3>
+					<p class="shift-left">
+						The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.
+					</p>
+					<ul>
+						<li>Schedule meetings for you</li>
+						<li>Excel automation to complete your work for you</li>
+						<li>Responds to emails on your behalf</li>
+						<li>Does all your work for you with our revolutionary AI technology</li>
+					</ul>
+				</section>
+				<section id="quote-area">
+					<section class="content-width">
+						<blockquote cite="#">
+							“Relaxr changed my life. I’ve been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work.”
+						</blockquote>
+						<h3>- Amanda, Intuit</h3>
+					</section>
+				</section>
+				<section id="sign-up">
+					<a href="#">Sign up Now!</a>
+				</section>
+		</main>
 			<footer>
-				&copy; Footer Goes Here
+				<h1>Relaxr</h1>
+				<ul>
+					<li><img src="images/twitter.png" alt="twitter logo" /></li>
+					<li><img src="images/facebook.png" alt="facebook logo" /></li>
+				</ul>
+				<p>
+				&copy; 2015. Relaxr
+			</p>
```

Comment:
:hammer: Indent this guy properly just to maintain best practices and make your code super-readable. Otherwise, excellent job on the indentation throughout.